### ISSUE TITLE
Installed but does not play audio

### ISSUE THREAD

**Issue by DylanLSpradlin** [AlexPresso/MediaServer-ffmpeg-patcher#3]
I installed the patcher but video station will still not play videos with the audio formats. Tried with truehd and dts

EDIT: I see now that I have an error for not having ffmpeg. It this a problem because I am using DSM7.2?

![image](https://github.com/AlexPresso/MediaServer-ffmpeg-patcher/assets/88327941/df5a6983-8d82-48ec-846b-4c77c45b7f26)

![image](https://github.com/AlexPresso/MediaServer-ffmpeg-patcher/assets/88327941/6d80d2d4-e513-4ae7-9155-aa933cc19a99)

**Comment by AlexPresso** [comment]
Hi,
You're not using the latest version of the patcher, which procedure did you follow?

**Comment by AlexPresso** [comment]
Aaaah no sorry, you're not using the right patch ;)

The one for Videostation is here : https://github.com/AlexPresso/VideoStation-FFMPEG-Patcher

**Comment by AlexPresso** [comment]
This one (MediaServer) has not been updated for a while, it's not maintained anymore and I will remove it soon

**Comment by DylanLSpradlin** [comment]
Replying here because this is where the thread is. I tried installing the one you linked. Similar scenario mentioning that ffmpeg is not installed. I see it running as a service though on the synology task manager. 

![image](https://github.com/AlexPresso/MediaServer-ffmpeg-patcher/assets/88327941/277156d2-f666-4b0e-97ad-1e9a05f6cc65)

This is confusing because when I play a video with a supported audio format, I see this in the synology task manager. 
![image](https://github.com/AlexPresso/MediaServer-ffmpeg-patcher/assets/88327941/340609ca-2694-4702-a563-9e4dd5f5f23e)


I found a video for this that shows that ffmpeg is installed in package center. This does not seem to be an optional install for DSM7.2 Is there a way to install this manually? Sorry I'm new to the whole Synology thing.
